docs/source/teams/index.rst

```diff
@@ -144,6 +144,7 @@ pages on this site apply to Teams deployments as well.
     FiftyOne Teams App <teams_app>
     Management SDK <management_sdk>
     Plugins <teams_plugins>
+    Data Lens <data_lens>
```

Comment:
Probably need product to help us say where to put this in the TOC, as it's starting to get quite long